<launch>
    <arg name="method" default="teb" doc="method type [teb, eband, dwa]"/>
    <!-- Run the global path planner -->
    <node pkg="simple_planner" type="global_planner" name="global_planner" respawn="true" output="log" >
        <param name="global_frame" value="map"/>
        <param name="plan_frequency" value="30"/>
        <param name="point_num" value="5"/>   
        <param name="waypoint_distance" value="0.4"/>
    </node>

    <!-- Run the local trajectory planner -->
 

    <node pkg="simple_planner" type="local_planner" name="local_planner" respawn="true" output="screen" >
        <param name="global_frame" value="odom"/>
        <param name="plan_frequency" value="30"/>
        <param name="max_speed" value="0.5"/>
        <param name="max_angle_diff" value="60"/>
        <param name="p_coeff" value="2"/>
        <param name="goal_tolerance" value="0.05"/>
        <param name="prune_ahead_distance" value="0.1"/>
    </node>


</launch>
